dialect/oracledialect/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/uptrace/bun/dialect/oracledialect
 
-go 1.22.0
+go 1.23
```

Comment:
Dont upgrade go version, we support go >= 1.22

Reply:
But it was already broken in https://github.com/uptrace/bun/pull/926 and merged into `master`: all `go.mod` files were updated and `1.22` was removed from `.github/workflows/build.yml`

Reply:
Thanks, i just saw it.